.github/workflows/test-and-demo.yaml

```diff
@@ -94,13 +87,16 @@ jobs:
           node_modules
           */node_modules
         key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
-    - name: Restore artifacts
+    - name: Cache storybook
       uses: actions/cache@v2
+      id: storybook-cache
       with:
         path: |
-          */dist
-        key: ${{ runner.os }}-${{ hashFiles('*/dist') }}
+          docs/dist
+          vue-components/storybook-static
+        key: ${{ runner.os }}-${{ hashFiles('tokens/properties/**/*', 'vue-components/stories/**/*', 'vue-components/src/**/*') }}
```

Comment:
could we also add a hash for the package-lock files here? So that the storybooks are also rebuild if some dependencies change. E.g. we are updating storybook 6.1.1 (which we should do)

Also, shouldn't this include the stories in `docs`?

And maybe the `.storybook` directories

After all these are included, I'm not that sure that there will be a lot PRs that don't touch the cache :thinking:

Reply:
Sure, will do on Monday